### DISCUSSION TITLE
Hi from darkshapes

### DISCUSSION THREAD

**Discussion by exdysa** [civitai/CivitUI#8] [Ideas]
We love what you did with the CivitUI look! In fact, we enjoyed it so much much we forked it, packed it into ComfyUI and tried to get it merged into main. That ended up sidelined on their end unfortunately, so we spun off our own system. Anyway, I remember there was a CivitAI model downloader component that was built in to the CivitUI client. I'm sure you're very busy, but I was wondering if you would be interested in spinning that off into a modular version as an extension to our new system, Shadowbox? I know it would look really cool if you did. Regardless, I wanted to let you know we appreciate your work on CivitUI and we're doing our best to give it new life. Thanks!

**Comment by JustMaier** [comment]
Hey sounds like a cool idea. I have a hard time keeping up with the various messaging channels. If you happen to be on Discord, let's connect so we can hash out the idea a bit more. You can catch me there at @justmaier
